k8s/namespaces/wa/wa-task-monitor/wa-task-monitor.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/wa/task-monitor:prod-b0f1dbc-20211006150635
+      image: hmctspublic.azurecr.io/wa/task-monitor:prod-b079fc5-20211015124206
```

Comment:
To enable image automation can you change it to `image: hmctspublic.azurecr.io/wa/task-monitor:prod-b079fc5-20211015124206 #{"$imagepolicy": "flux-system:wa-task-monitor"}`